flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/util/JobConfUtilsTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.connectors.hive.util;
+
+import org.apache.hadoop.io.Text;
+import org.apache.hadoop.mapred.JobConf;
+import org.apache.hadoop.security.Credentials;
+import org.apache.hadoop.security.UserGroupInformation;
+import org.apache.hadoop.security.token.Token;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+
+/** Test for {@link JobConfUtils}. */
+public class JobConfUtilsTest {
```

Comment:
Since the credentials are added in `HiveDynamicTableFactory`, it's better to add the test in `HiveDynamicTableFactoryTest` to verify this behavior.

Reply:
Do I need to remove JobConfUtilsTest test?

Reply:
Yeah. I think we don't need it.

Reply:
But it's hard to test in `HiveDynamicTableFactoryTest`. I have to get the `HiveTableSource` to test its jobConf credentials, however `HiveTableSources` don't have `getJobConf` method. 
Do you have some ideas on it?

Reply:
Yes, a getter would be fine